core/src/main/java/org/elasticsearch/action/admin/indices/delete/DeleteIndexResponse.java

```diff
@@ -48,4 +51,34 @@ public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         writeAcknowledged(out);
     }
+
+    public static DeleteIndexResponse fromXContent(XContentParser parser) throws IOException {
```

Comment:
We need to add unit tests for this parsing method, similar to what we did in `IndexResponseTests`, `GetResponseTests` etc.